lib/Conversion/TritonGPUToLLVM/HistogramOpToLLVM.cpp

```diff
@@ -146,6 +143,11 @@ static SmallVector<Value> computeCrossWarpHistogram(
     Value val = b.load(i32_ty, sharedMemPtr);
     histogramValues.push_back(val);
   }
+  for (auto i = 0; i < histogramValues.size(); ++i) {
+    histogramValues[i] =
+        b.sdiv(histogramValues[i], b.i32_val(replicationFactor));
+  }
+
```

Comment:
do we have to divide twice? Can we do it only once? Fine to skip if it makes the code more complicated